src/System.Device.Gpio/System/Device/Pwm/55-pwm.rules

```diff
@@ -0,0 +1,4 @@
+SUBSYSTEM=="pwm*", PROGRAM="/bin/sh -c '\
```

Comment:
I don't really like this part, I don't believe that we want our libraries copying files around that will change ownership of files. I much rather require root permissions in order to run our library (which is kind of already a requirement for gpio and other parts of the library anyways) than trying to bypass security that will affect the state of your PI/device aftewards.

Reply:
I think this should be described only in the README.md. I don't want to get into changing OS configuration path

Reply:
I just provided the file, the code does nothing else with it. You have to copy it manually (usually to /etc/udev/rules.d). Should be described in either case, though. 
I think that's the last part that currently require root rights on the Pi. Gpio access no longer does.

Reply:
Depends on what part of Gpio and which driver you are using. I agree with @krwq that it is fine to have this on the README or a wiki, but not checked-in as an extra file to the code.

Reply:
also keep in mind this particular file was copied from GPL v2 repo (at least per your comment in the issue) which means we cannot use it directly - although you can probably use something similar based on what I've pasted in the issue